repos/coreclr.proj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <BuildArguments>$(Platform) $(Configuration) skiptests -PortableBuild=$(PortableBuild)</BuildArguments>
     <BuildArguments Condition="'$(OS)' != 'Windows_NT'">$(BuildArguments) msbuildonunsupportedplatform</BuildArguments>
+    <BuildArguments Condition="'$(UseSystemLibraries)' != 'false'">$(BuildArguments) cmakeargs -DCLR_CMAKE_USE_SYSTEM_LIBUNWIND=TRUE</BuildArguments>
```

Comment:
Normally we do `... == 'true'` in MSBuild, is there a reason to take more than `true` for this arg in particular?